### PR TITLE
chore: fix bootstrap scripts

### PR DESCRIPTION
**Pull request by drduhe**
**Issue #, if available:** n/a

### Notes
* Fixing bootstrap scripts to run correctly on their corresponding operating systems.

### Testing
Created fresh EC2 instances for both Ubuntu and AL2, ran scripts, validated installations. 

Before you submit a pull request, please make sure you have to following:

- [x] Your code contains tests that cover all new code and changes
- [x] All new and existing tests passed
- [x] I have read the [Contributing Guidelines](https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/blob/main/CONTRIBUTING.md) and agree to follow the [Code of Conduct](
https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/blob/main/CODE_OF_CONDUCT.md))

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: scripts/ec2_bootstrap_al2.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    && git-lfs pull
+
+chmod -R 777 guidance-for-overhead-imagery-inference-on-aws
+
+# Clone submodules from HTTPS instead of SSH (since git ssh keys are not available yet)
+cd guidance-for-overhead-imagery-inference-on-aws \
+    && sed -i 's/git@github.com:/https:\/\/github.com\//g' .gitmodules \
+    && git submodule update --init --recursive
+
+echo "--------------------- EC2 Bootstrap complete ---------------------"
```
Comment (RanbirAulakh) [inline scripts/ec2_bootstrap_al2.sh:73]: Add extra line

File: scripts/ec2_bootstrap_al2.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws.git \
+    && cd guidance-for-overhead-imagery-inference-on-aws \
+    && git-lfs pull
+
+chmod -R 777 guidance-for-overhead-imagery-inference-on-aws
+
+# Clone submodules from HTTPS instead of SSH (since git ssh keys are not available yet)
+cd guidance-for-overhead-imagery-inference-on-aws \
+    && sed -i 's/git@github.com:/https:\/\/github.com\//g' .gitmodules \
+    && git submodule update --init --recursive
```
Comment (RanbirAulakh) [inline scripts/ec2_bootstrap_al2.sh:71]: Replace all `guidance-for-overhead-imagery-inference-on-aws` with `guidance-for-processing-overhead-imagery-on-aws`

File: scripts/ec2_bootstrap_ubuntu.sh [hunk 3]
```diff
@@ -73,26 +65,12 @@ unzip -o awscliv2.zip
...
-# Install Python packages
-su - "$USERNAME" -c "pip3 install geojson pytest boto3"
+snap install docker
 
 # Clone primary AWS repo and configure git
-su - "$USERNAME" -c "git clone https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws.git && cd guidance-for-overhead-imagery-inference-on-aws && git-lfs pull && sed -i 's/git@github.com:/https:\/\/github.com\//g' .gitmodules && git submodule update --init --recursive"
-
+git clone https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws.git
+chmod -R 777 guidance-for-overhead-imagery-inference-on-aws
+cd guidance-for-overhead-imagery-inference-on-aws || exit
```
Comment (RanbirAulakh) [inline scripts/ec2_bootstrap_ubuntu.sh:73]: Replace all `guidance-for-overhead-imagery-inference-on-aws` with `guidance-for-processing-overhead-imagery-on-aws`

File: scripts/ec2_bootstrap_al2.sh [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+#
+# Copyright 2024 Amazon.com, Inc. or its affiliates.
+#
+
+#############################################
+# EC2 Bootstrap Script for OSML Demo
+#
+# Installs all necessary dependencies on a fresh EC2 instance to deploy the OSML demo.
+# Known good configuration:
+#     - OS: Ubuntu 22.04 LTS (AMI ID: ami-08116b9957a259459)
```
Comment (RanbirAulakh) [inline scripts/ec2_bootstrap_al2.sh:11]: Update the OS and AMI-ID

File: scripts/ec2_bootstrap_ubuntu.sh [hunk 5]
```diff
@@ -73,26 +65,12 @@ unzip -o awscliv2.zip
...
+git-lfs pull && sed -i 's/git@github.com:/https:\/\/github.com\//g' .gitmodules
+git submodule update --init --recursive
```
Comment (jtblack-aws) [inline scripts/ec2_bootstrap_ubuntu.sh:75]: match these lines with AL2 lines (or vice versa)

**Review by RanbirAulakh — APPROVED** [review]

**Review by natekruse-aws — APPROVED** [review]
LGTM

**Pull request merged**